e2e/helpers/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { Page, expect } from '@playwright/test'
+
+// Use after initially navigating to baseURL
+// ...to give SWC time to compile, if necessary
+export async function waitForInitialPageLoad(page: Page) {
+  await expect(page.getByRole('heading', { name: 'Loading…' })).not.toBeVisible(
+    { timeout: 2 * 60 * 1000 }, // ...wait 2 minutes if necessary
+  )
+}
+
+export async function navigateToHomepageIfPageHasNotBeenLoaded(page: Page) {
+  if (page.url() === 'about:blank') {
+    // Navigate to homepage, so that localStorage and SRC are available
+    await page.goto('/')
+    await waitForInitialPageLoad(page)
+  }
+}
```

Comment:
The first test that runs (`auth.setup`) creates a test user by evaluating the page to call SRC's doPost. I'm wondering if the flaky test in [this run](https://github.com/Sage-Bionetworks/SynapseWebClient/actions/runs/6113921796/job/16594399197) was related to evaluating the page before the initial page load finished. I added an await for the initial page load here to hopefully avoid that issue in the future.